### PR TITLE
Applications brief

### PR DESCRIPTION
**Pull request by dannyroberts**
This is just cleanup; replaces direct calls to a view with calls to a dbaccessors

### CODE REVIEW

File: corehq/apps/reminders/util.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
 from datetime import datetime, time
 from corehq import privileges
-from corehq.apps.app_manager.dbaccessors import get_app
+from corehq.apps.app_manager.dbaccessors import get_app, get_brief_apps_in_domain, \
```
Comment (dimagimon) [inline corehq/apps/reminders/util.py]: F401 'get_brief_apps_in_domain' imported but unused

**Comment by dannyroberts** [comment]
:white_check_mark:

**Comment by benrudolph** [comment]
nice cleanup!

**Comment by dannyroberts** [comment]
Thanks!

File: corehq/apps/dashboard/models.py [hunk 2]
```diff
@@ -317,13 +317,7 @@ def total(self):
...
     @memoized
     def applications(self):
-        key = [self.request.domain]
-        return Application.get_db().view(
-            'app_manager/applications_brief',
-            reduce=False,
-            startkey=key,
-            endkey=key+[{}],
-        ).all()
+        return get_brief_apps_in_domain(self.request.domain)
```
Comment (dannyroberts) [inline corehq/apps/dashboard/models.py]: @benrudolph oops this change wasn't quite equivalent and broke something. Fixed here: https://github.com/dimagi/commcare-hq/pull/9953

**Pull request merged**